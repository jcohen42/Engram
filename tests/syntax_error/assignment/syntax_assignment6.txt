Begin program:
	is greater than
	Set y equal to 3.
	Output x modulo y.
	Output y modulo x.
End program.