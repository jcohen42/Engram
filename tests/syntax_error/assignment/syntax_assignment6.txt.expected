Syntax error encountered on line 2!